### ISSUE TITLE
Team listing is outdated; images on individual pages not displayed.

### ISSUE THREAD

**Issue by jdavcs** [galaxyproject/galaxy-hub#1558]
Two minor, non-urgent items:

1. The team page at  https://galaxyproject.org/galaxy-team/ is outdated (linked from the Galaxy Community page)
2. The image display on team member pages is broken (https://galaxyproject.org/people/john-davis/ displays this: 
    `![](/galaxy-team/johndavis.jpg)`

**Comment by NickSto** [comment]
Yeah, that page is so outdated, it shouldn't be linked to so prominently, and eventually we should remove or revamp it entirely.

As for the broken images, that's a symptom of moving to a modern Markdown parser, not anything specific to team member pages. It's an easy, but manual fix: #1562. But we're actively working on the Community Directory project (#1551), which will change how all the personal pages display, so we'll probably wait on that before manually fixing all those images.

**Comment by jdavcs** [comment]
Closed in favor of #2421
